src/Directory.Build.props

```diff
@@ -1,7 +1,3 @@
 <Project>
   <Import Project="..\Directory.Build.props" />
```

Comment:
Just a reminder that you can remove this file if you wanted too. Again, if you do just make sure that you make sure people upstack are importing the right Directory.Build.props from the root.

Reply:
Right, that's why I'm leaving it here. Otherwise I'd have to update each csproj to refer to the root Directory.Build.props. This way any new proj added to the repo should just inherit the props